tests/Unit/Helper/MiscHelperTest.php

```diff
@@ -33,4 +33,12 @@ public function getInputOutputForSplitSecondsMicroseconds()
             array('3.123456', array(3, 123456)),
         );
     }
+
+    public function testHexDump()
+    {
+        $res = MiscHelper::hexdump('FM', 'hmtlOutput' == false, 'uppercase' == false, 'return' == true);
```

Comment:
Sure, you can use your method because it's test code. It has the side effect of setting the $htmlOutput, $uppercase and $return variable, though, which is why it's not a good practice in the general case.

The method I use is only valid for "documenting" boolean arguments passed to external methods. 'x'==false evaluates to false, 'x'==true evals to true, for any string. Since it doesn't work for numerical arguments, you can argue that it's not very generalizable either...